iot-hub/Tutorials/Routing/SimulatedDevice/Program.cs

```diff
@@ -60,23 +56,30 @@ private static void Main(string[] args)
                 //  http://docs.microsoft.com/azure/iot-hub/tutorial-routing
 
                 Console.WriteLine("Routing Tutorial: Simulated device\n");
-                s_deviceClient = DeviceClient.Create(s_iotHubUri, new DeviceAuthenticationWithRegistrySymmetricKey(s_myDeviceId, s_deviceKey), TransportType.Mqtt);
-                SendDeviceToCloudMessagesAsync();
+                s_deviceClient = DeviceClient.Create(s_iotHubUri, 
+                  new DeviceAuthenticationWithRegistrySymmetricKey(s_myDeviceId, s_deviceKey), TransportType.Mqtt);
+
+                var cts = new CancellationTokenSource();
```

Comment:
using (requires disposing)

Reply:
`using var cts = new CancellationTokenSource();`

If that doesn't work, (there is a version requirement), you can give it a body:

```csharp
using (var cts = new CancellationTokenSource())
{
    /// ...
}
```